src/Serilog/LoggerConfigurationExtensions.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright 2013-2015 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.ComponentModel;
+using Serilog.Configuration;
+using Serilog.Core;
+using Serilog.Debugging;
+using Serilog.Enrichers;
+using Serilog.Events;
+using Serilog.Formatting.Display;
+using Serilog.Formatting.Raw; 
+
+#if PROCESS
+using System.Diagnostics;
+#endif
+
+
+#if APPSETTINGS
+using Serilog.Settings.AppSettings;
+#endif
+
+namespace Serilog
+{
+    /// <summary>
+    /// Extends <see cref="LoggerConfiguration"/> to add Full .NET Framework
+    /// capabilities.
+    /// </summary>
+    public static class LoggerConfigurationExtensions
+    {
+        //TODO: Need to confirm this is the best location for this default.  Used in File, Trace, RollingFile.  Do we move this to each sink?
+        public const string DefaultOutputTemplate = "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception}";
```

Comment:
I'd definitely favour moving these - duplication should be fine, the reuse by different sinks is only opportunistic/coincidental.